Main.java

```diff
@@ -9,7 +11,23 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+  //creates a scanner for user input
+  Scanner input = new Scanner(System.in); 
+
+  //prompt user for their numbers 
+  System.out.println("Please enter an integer");
+  int integer = input.nextInt(); 
+  System.out.println("please enter your second integer");
+  int secondInteger = input.nextInt(); 
+
+  //determine how the numbers will ascend in order
+  if(integer >= secondInteger){
+    System.out.println(" your numbers in ascending order are " + secondInteger + "," +integer);
+  }else{ 
+    System.out.println(" your numbers in ascending order are " + integer+ "," + secondInteger);
+  }
+    
```

Comment:
Watch the indentation. Since all of this code is inside the main method, it should all be TABBED over one level so that it appears "inside" the main method. Otherwise, good work!